packages/require/lib/types.ts

```diff
@@ -0,0 +1,60 @@
+import TruffleConfig from "@truffle/config";
+import Resolver from "@truffle/resolver";
+
+export interface RequireOptions {
+  file: string;
+  config: TruffleConfig;
+}
+
+export interface ExecOptions {
+  contracts_build_directory: string;
+  /**
+   *  Path to file to execute. Must be a module that exports a function.
+   */
+  file: string;
+
+  /**
+   * Arguments passed to the exported function within file. If a callback is not
+   * included in args, exported function is treated as synchronous.
+   */
+  args: any[];
```

Comment:
I kept this from the original code, but I couldn't see anywhere that this was actually referenced. Should I just remove it?

Reply:
/shrug

Reply:
I'll remove it and any other stuff that isn't referenced